__main__.py

```diff
@@ -6,6 +6,6 @@
 
```

Comment:
Just a note: I believe we need to switch for 80 chars per line to 100.

And we use this header now:
`# vim: ts=4 sw=4 et ai si
`
Instead, we should use this:

`# vim: ts=4 sw=4 tw=100 et ai si
`
Would you care enough to do this too? Thanks in advance!

Reply:
Following the [black code style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length), I set the `textwidth` to `88`.